document workflow log files (#750)
cylc/cylc-doc: src/user-guide/running-workflows/scheduler-log-files.rst

.. _Scheduler Logs:
.. _user-guide.log_files:

Workflow Logs
=============

Cylc produces a number of log files which record information about the

> wortkflow, how it was installed, what tasks have run, their outcomes and more.

Reviewer: ```suggestion
workflow, how it was installed, what tasks have run, their outcomes and more.
```

Wortflow? Is that a brewing term?

> Cylc produces a number of log files which record information about the
> wortkflow, how it was installed, what tasks have run, their outcomes and more.

Reviewer: I think it's worth ordering this sentence by the order in which Cylc logs stuff.
```suggestion
Cylc produces log files which record information how the workflow was
installed, configured when started or changed, and about tasks run.
```

Author: Missing an "about" between "information" and "how"

Author: Sorry, "and about tasks run" doesn't make sense.

"and the tasks that have run and their outcomes"?

Reviewer: Have edited my proposed change. I agree.
